src/backend/distributed/planner/merge_planner.c

```diff
@@ -600,7 +626,8 @@ MergeQualAndTargetListFunctionsSupported(Oid resultRelationId, Query *query,
 		}
 
 		if (targetEntryDistributionColumn &&
-			TargetEntryChangesValue(targetEntry, distributionColumn, joinTree))
+			TargetEntryChangesValue(targetEntry, distributionColumn, (Node *) query,
+									CMD_MERGE))
```

Comment:
Just noting that we're explicitly passing `CMD_MERGE` at this point (line 630). Will `commandType` also have the value `CMD_MERGE` ? Can we pass `commandType` here, and achieve the same effect ? I guess I'm curious if there's a reason to explicitly pass `CMD_MERGE` here..

Reply:
With dummy FromTree, we don't need this anymore